### DISCUSSION TITLE
Exporting Spline design as React code

### DISCUSSION THREAD

**Discussion by anyakath** [splinetool/react-spline#143] [Q&A]
Followed all the tutorials but I somehow keep getting this error when I try to integrate my spline design in my react website:

./node_modules/@splinetool/react-spline/dist/react-spline.js 834:62
Module parse failed: Unexpected token (834:62)
You may need an appropriate loader to handle this file type.
         P = new F(function (y) {
           y.forEach(function (s) {
             var _ref2 = (s == null ? void 0 : s.contentRect) ?? {},
              D = _ref2.left,
             x = _ref2.top,

Can't get to the root of this. Any help/suggestions would be really appreciated.

**Comment by marcofugaro** [comment]
You need to use react-scripts version 5.0.0 or greater

**Reply by anyakath** [reply]
Thank you so much! works now!
